include/mata/nfa.hh

```diff
@@ -561,7 +538,37 @@ public:
      * @brief Construct a new explicit NFA from other NFA.
      */
     Nfa(const Mata::Nfa::Nfa& other) = default;
+
+    Nfa(Mata::Nfa::Nfa&& other) noexcept
+        : delta{ std::move(other.delta) }, initial{ std::move(other.initial) }, final{ std::move(other.final) },
+          alphabet{ other.alphabet }, attributes{ std::move(other.attributes) },
+          m_num_of_requested_states{ other.m_num_of_requested_states } {
+        other.delta = {};
+        other.initial = {};
+        other.final = {};
+        other.attributes = {};
```

Comment:
Isn't this done in the `std::move` operations?

Reply:
Solved, see comment above.